index.js

```diff
@@ -3,30 +3,69 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+  const result = []
+  for (let i = 0; i <= rolls.length - 1; i++) {
+    rolls.push(i);
+    return true;
+
+
+  }
+  return false;
+
+
+}
+//^---1 out of 3 checks,must return 2 different true statement but only passing false statement 
+//BUT when i leave the parameter in, no test pass
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  const result = []
+  for (let i = 0; i <= rolls.length - 1; i++) {
+    rolls.push(i)
 
+    return value;
+  }
+  return null;
+}
```

Comment:
Take a look at what the instructions are asking you to return - either null or a number. Since it will only be a number if the value passed is in rolls, then your default value for `result` should be `null`. 

Then in your loop you have to check each element to see if it contains the value that was passed.